cmd/cloudtrail/write-events.go

```diff
@@ -0,0 +1,392 @@
+package cloudtrail
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"slices"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail/types"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/osdctl/pkg/osdCloud"
+	"github.com/openshift/osdctl/pkg/osdctlConfig"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+// Configuration struct for parsing configuration options
+type Configuration struct {
+	Ignore struct {
+		Users []string `mapstructure:"users_with"`
+	} `mapstructure:"ignore"`
+}
```

Comment:
We can make the configuration more descriptive. If I add this to my osdctl configuration file, I would not know this is related to the cloudwatch cmmands.